docs/reference/pip_install.rst

```diff
@@ -16,14 +16,17 @@ Description
 .. pip-command-description:: install
 
 
-.. _`Requirements File Format`:
-
 Installation Order
 ++++++++++++++++++
 
-Pip installs dependencies before the things that depend on them. In the event
-of a dependency cycle, the first encountered member of the cycle is installed
-last.
+As of v6.1.0, pip installs dependencies before their dependents, i.e. in
```

Comment:
I think this should still document the cycle breaking behaviour. It could be more vague if we don't want to pin ourselves down to a particular behaviour, but it should still say that cycles are handled by breaking them.